src/components/UI/Dropdown/index.js

```diff
@@ -1,19 +1,73 @@
-import React from "react";
-import i18next from "i18next";
-// Styles
-import { Container } from "./dropdown.styled";
+import React, { useState } from "react";
+import styled from "styled-components";
+import {IoIosArrowDown} from 'react-icons/io';
 
-export function Dropdown() {
-  const handleLanguageChange = (lang = "en") => {
-    i18next.changeLanguage("sr");
-  };
+
+export function Dropdown(props) {
+  const [isOpen, setIsOpen] = useState(false);
+  const toggling = () => setIsOpen(!isOpen);
+  const [selectedOption, setSelectedOption] = useState(props?.list[0]);
```

Comment:
If we are checking if list exists in props, why aren't we checking if the first element exists in the list?